libgstd/gstd_log.c

```diff
@@ -34,20 +34,21 @@ GST_DEBUG_CATEGORY (gstd_debug);
 #define GSTD_DEBUG_PREFIX "gstd"
 #define GSTD_DEBUG_LEVEL "WARNING"
 
-static void
-gstd_log_proxy (GstDebugCategory * category, GstDebugLevel level,
-    const gchar * file, const gchar * function, gint line, GObject * object,
-    GstDebugMessage * message, gpointer user_data) G_GNUC_NO_INSTRUMENT;
-
 static const gchar *gstd_log_get_gstd_default (void);
 static const gchar *gstd_log_get_gst_default (void);
 static gchar *gstd_log_get_filename (const gchar * filename,
     const gchar * default_filename);
 
-static FILE *_gstdlog = NULL;
-static FILE *_gstlog = NULL;
-static gchar *gstd_filename;
-static gchar *gst_filename;
+static void
+gstd_log_proxy (GstDebugCategory * category, GstDebugLevel level,
+    const gchar * file, const gchar * function, gint line, GObject * object,
+    GstDebugMessage * message, gpointer user_data)
+    G_GNUC_NO_INSTRUMENT;
```

Comment:
Can you try changing this to:
```
   GstDebugMessage * message, gpointer user_data) G_GNUC_NO_INSTRUMENT;
```
This is to keep the `;` in the same lane as the last line of the function. Otherwise this is causing the incorrect indentation on the next lines

Reply:
If the indentation tool forces this ordering order don't force this

Reply:
That change is made by the indentation script